### PR TITLE
Adds initial security documentation

### PR DESCRIPTION
**Pull request by michael-fielding**
First version of security architecture documentation

Review for accuracy of content as well as grammar/spelling issues

### CODE REVIEW

File: documentation/architecture/images/High-Level-Viewpoint.png
Comment (jrabbott) [inline documentation/architecture/images/High-Level-Viewpoint.png:1]: Is the web app session data store (cache) supposed to be part of platform? 

Do we need to include the SQL db?

What's the difference between Monitoring and App Insights here?

File: documentation/architecture/images/Logical Architecture.png
Comment (jrabbott) [inline documentation/architecture/images/Logical Architecture.png:1]: Might be worth a discussion on this one? Quite a few bits I'm not clear on?

File: documentation/architecture/images/Security Architecture.png
Comment (jrabbott) [inline documentation/architecture/images/Security Architecture.png:1]: Same here - hight be worth a discussion on this one? Quite a few bits I'm not clear on?

File: documentation/architecture/images/High-Level-Viewpoint.png
Comment (michael-fielding) [reply documentation/architecture/images/High-Level-Viewpoint.png:1]: The intent is to have logging as more maintenance-based (error logging) and app insights to give us more of the BI/MI type info

I suppose we should include the SQL DB, although currently I envisage it being temporary. That of course may change as we continue to iterate

IS the cache supposed to be part of the web side? If so I can change it

Comment (jrabbott) [reply documentation/architecture/images/High-Level-Viewpoint.png:1]: Monitoring is covered by App Insights - we log everything there.

**Review by jrabbott — APPROVED** [review]

**Pull request merged**